tailwind.config.js

```diff
@@ -75,42 +74,43 @@ module.exports = {
           lighter: "#C79D5E",
         },
       },
-      backgroundImage: {
-        abyss: "url('/public/images/scale-of-suffering/abyss.jpg')",
-        "dirt-road": "url('/public/images/population-dynamics/dirt-road.jpg')",
-        "ant-hill": "url('/public/images/how-to-help/ant-hill.png')",
-        forest: "url('/public/images/how-to-help/forest.jpg')",
-        frog: "url('/public/images/organizations/frog.png')",
-        hedgehog:
-          "url('/public/images/common-objections/grass-and-hedgehog.png')",
-        leaves: "url('/public/images/how-to-help/leaves.png')",
-        mud: "url('/public/images/organizations/mud-bg-color.jpg')",
-        sand: "url('/public/images/types-of-suffering/sand-bg-color.jpg')",
-        savanna: "url('/public/images/introduction/savanna.jpg')",
-        "savanna-md": "url('/public/images/introduction/savanna-md.jpg')",
-        woods: "url('/public/images/conclusion/woods.jpg')",
-        "woods-md": "url('/public/images/conclusion/woods-md.jpg')",
-        sky: "url('/public/images/videos/sky.jpg')",
-        "resource-section": "url('/public/images/other-resources/section.png')",
-        "resource-section-active":
-          "url('/public/images/other-resources/section-active.png')",
-        "resource-section-flipped":
-          "url('/public/images/other-resources/section-flipped.png')",
-        "resource-section-active-flipped":
-          "url('/public/images/other-resources/section-active-flipped.png')",
-        "tree-and-birds":
-          "url('/public/images/organizations/tree-and-birds.png')",
-      },
       fontFamily: {
         sans: ["Poppins", "sans-serif"],
         brand: ["The Bold Font", "sans-serif"],
       },
-      boxShadow: {
-        glow: "0 0 3px 1px #fff",
-        "glow-lg": "0 0 10px 1px #fff",
+      height: () => ({
```

Comment:
I actually think we can remove these height utilities. Don't think they are used anymore.